### PR TITLE
Containerise ModelPolisher

### PR DESCRIPTION
**Pull request by codekaust**
Fix issue #38.

### CODE REVIEW

**Comment by mephenor** [comment]
Just two small things:
- Adding `--user $(id -u):$(id -g)` to the `docker-compose run` command will produce output with correct ownership
- There are still the  `|| true`s that we should remove, if possible. This should directly work for adb and will work for biggdb, if postgresql used for the second one is downgraded to 9.6.2

File: src/main/java/edu/ucsd/sbrg/bigg/SBMLPolisher.java [hunk 1]
```diff
@@ -50,8 +50,8 @@
 import de.zbit.kegg.AtomBalanceCheck.AtomCheckResult;
 import de.zbit.util.progressbar.AbstractProgressBar;
 import de.zbit.util.progressbar.ProgressBar;
-import edu.ucsd.sbrg.util.SBMLFix;
-import edu.ucsd.sbrg.util.SBMLUtils;
+import main.java.edu.ucsd.sbrg.util.SBMLFix;
+import main.java.edu.ucsd.sbrg.util.SBMLUtils;
```
Comment (draeger) [inline src/main/java/edu/ucsd/sbrg/bigg/SBMLPolisher.java]: This is not correct. Moving the files from `/src/` to `/src/main/java` doesn't mean we have to change all the packages. Please change this again. The package is still `edu.ucsd.sbrg.util...`. The source code is now just in a different folder.

File: src/main/java/edu/ucsd/sbrg/cobra/COBRAparser.java [hunk 2]
```diff
@@ -1,9 +1,9 @@
 /**
  *
  */
-package edu.ucsd.sbrg.cobra;
+package main.java.edu.ucsd.sbrg.cobra;
```
Comment (draeger) [inline src/main/java/edu/ucsd/sbrg/cobra/COBRAparser.java]: See above. The package is wrong.

Comment (codekaust) [reply src/main/java/edu/ucsd/sbrg/cobra/COBRAparser.java]: I have made the changes, I think these changes were done automatically by IDE.

**Comment by codekaust** [comment]
> Just two small things:
> 
> * Adding `--user $(id -u):$(id -g)` to the `docker-compose run` command will produce output with correct ownership
> * There are still the  `|| true`s that we should remove, if possible. This should directly work for adb and will work for biggdb, if postgresql used for the second one is downgraded to 9.6.2

It is working if we use `postgres:9.6.2` as base image for biggdb image, but also we cannot use `postgres:9.6.2` base for adb.
It gives the following error:
```
pg_restore: [archiver] unsupported version (1.13) in file header
```
This is probably because of using an old version of pg_restore that doesn't support restoring the provided dump file.

So, we can keep base as `11.4` for adb and `9.6.2` for biggdb, but yes it then downloades one extra image(postgres:9.6.2), increasing the total downloads for ModelPolisher by about 270mb. Changes in commit c2bb729640faa2fba4b1efb9eb4517fc031996ca

**Pull request merged**